lesson02/assignment/css/style.scss

```diff
@@ -1,6 +1,49 @@
-// Import partials here
-@import 'base.normalize';
-@import 'settings.variables';
+// Import partials here @import 'base.normalize'; 
 @import 'settings.responsive';
-@import 'components.content';
+@import 'settings.variables';
+//@import 'components.content';
+
+// this has to come before -leftCard and _rightCard
+@mixin border-radius($r) {
+	-webkit-border-radius: $r;
+	-moz-border-radius: $r;
+	-ms-border-radius: $r;
+	border-radius: $r;
+}
+
+// card border
+.mkcard {
+	margin-top: 100px;
+	margin-left: 100px;
+    border: 20px solid $gray;
+    width: 540px;
+    height: auto;
+    background: $white;
+}
+
+// card container
+.card {
+    border: 10px solid $white;
+    width: 500px;
```

Comment:
The card breaks down well in mobile, but be mindful of setting a fixed width at a smaller screen size, you can also give it a `max-width: 500px; and width: 100%; ` which will never go beyond 500px but wont cause scrolling in smaller screens.